### ISSUE TITLE
Problems when training on 300W

### ISSUE THREAD

**Issue by zhang-zx** [facebookresearch/supervision-by-registration#42]
Thanks for your fantastic code. However, when I train the model using `./scripts/300W-DET.sh`, I can not get the NME 4.36 given in the paper. Is there anything missed  in the README?

**Comment by D-X-Y** [comment]
would you mind to share the log?

**Comment by zhang-zx** [comment]
[seed-93860-time-09-Apr-at-05-59-39.log](https://github.com/facebookresearch/supervision-by-registration/files/3057527/seed-93860-time-09-Apr-at-05-59-39.log)
This is the log during training.

**Comment by D-X-Y** [comment]
That is a little bit strange. From your log, the detection model didn't converge at all.
You can find my log at here #23 . Did you change any lines, because I see that your log show a total epoch of 200, while the default setting of my codes is 50?

Give me some time to re-run the codes to see what is the problem.

**Comment by zhang-zx** [comment]
I changed nothing but the total epochs.

**Comment by D-X-Y** [comment]
Sure, may I know your PyTorch version and Python version?

**Comment by zhang-zx** [comment]
<img width="703" alt="image" src="https://user-images.githubusercontent.com/21698324/55789397-db53e180-5aec-11e9-8eb9-bde3d67ae4f2.png">
